modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -162,6 +162,25 @@ definition {
 		}
 	}
 
+	macro editAndPublishRelatedEntryConfigInEditApplication {
```

Comment:
How about naming it to be addAndPublishFiltersAndSortingInEndpointConfiguration?